testutil/wait.go

```diff
@@ -69,8 +63,18 @@ func Timeout(original time.Duration) time.Duration {
 	return original * time.Duration(TestMultiplier())
 }
 
+func IsCI() bool {
+	_, ok := os.LookupEnv("CI")
```

Comment:
maybe this should be `ok || IsTravis() || IsAppveyor()` for completeness but every CI provider sets CI